packages/web/src/store/sagas.ts

```diff
@@ -227,5 +228,5 @@ export default function* rootSaga() {
     // Version refresh
     reloadSagas()
   )
-  yield all(sagas.map(fork))
+  yield all(sagas.map((saga) => spawn(sagaWithErrorHandler, saga)))
```

Comment:
Should we do the same on mobile?

Reply:
yea was planning to if we think this makes sense

Reply:
I'm wondering if there are certain sagas where this could cause havoc, like the ones that are meant to run once at app startup and then immediately do work to bootstrap and initialize. Worth considering whether those just just be allowed to fail and require a refresh.

Reply:
Yeah we should make sure that `waitForRead` and `waitForWrite` etc all still work on restart... I wonder if there's any other gotchas in our app. For the most part I think we're ok?

This does feel like one of those things that we definitely don't want to rely on and we should still try our best to have great error handling in each saga